### ISSUE TITLE
Use min versions of referenced resources

### ISSUE THREAD

**Issue by GoogleCodeExporter** [erhanselman/mobile-web-development-with-phonegap#36]
```
From @mkamoski's email to the mailing list:

Dear AppLaud Developer -

This is a suggestion.

I think that the project wizard should pick min versions of resources
by default, when such "min" versions exist.

Or, alternately, it should offer an option of either using the min-
version or the non-min-version of the resources.

For example, as of now, the project produces the following in the
"index.html" file...

<link rel="stylesheet" href="jquery.mobile/jquery.mobile-1.0b1.css"
type="text/css" charset="utf-8" />

<link rel="stylesheet" href="pgandjqm-style-override.css" type="text/
css" charset="utf-8" />

<script type="text/javascript" src="jquery.mobile/jquery-1.5.2.js"></
script>

<script src="jquery.mobile/jquery.mobile-1.0b1.js"></script>

...and I suggest that it should produce something like the following
instead...

<link rel="stylesheet" href="jquery.mobile/
jquery.mobile-1.0b1.min.css" type="text/css" charset="utf-8" />

<link rel="stylesheet" href="pgandjqm-style-override.min.css"
type="text/css" charset="utf-8" />

<script type="text/javascript" src="jquery.mobile/
jquery-1.5.2.min.js"></script>

<script src="jquery.mobile/jquery.mobile-1.0b1.min.js"></script>

...and that would be a bit more performant, etc.


```

Original issue reported on code.google.com by `paul.beu...@gmail.com` on 19 Jul 2011 at 6:50

**Comment by GoogleCodeExporter** [comment]
Original comment by `paul.beu...@gmail.com` on 19 Jul 2011 at 7:00
- Added labels: **Milestone-Release1.2.4**

**Comment by GoogleCodeExporter** [comment]
```
For 1.2.4 the default bundled jquery will be the min version. However, phonegap 
and jquery mobile will by default, stay as unminified to ease development.
```

Original comment by `paul.beu...@gmail.com` on 16 Aug 2011 at 9:39

**Comment by GoogleCodeExporter** [comment]
```
Fixed with 1.2.4
```

Original comment by `paul.beu...@gmail.com` on 19 Aug 2011 at 4:30
- Changed state: **Fixed**
